src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -590,9 +590,28 @@ public boolean hasNoOngoingBusinessProcess() {
     private List<CaseManagementOrderAdditional> caseManagementOrderAdditional;
     //general application order documents
     private final List<Element<CaseDocument>> generalOrderDocument;
```

Comment:
generalOrderDocument, dismissalOrderDocument, directionOrderDocument and hearingNoticeDocument . .cant this be removed ?

Reply:
> generalOrderDocument, dismissalOrderDocument, directionOrderDocument and hearingNoticeDocument . .cant this be removed ?

in GA case data, those 4 fields are still in use, for hold document at GA case data. the new added 4x4 fields are for hold documents at Civil case side